src/validator.js

```diff
@@ -471,4 +474,21 @@ export default class Validator {
       return ARCH_DEFAULT;
     }
   }
+
+  _markEmptyFiles(addonMetadata) {
+    var emptyFiles = [];
+
+    return this.io.getFiles()
+      .then((files) => {
+        for (let filename in files) {
+          if (files[filename].uncompressedSize === 0) {
```

Comment:
This API is different for the directory class [1].

Maybe we should add uncompressedSize as an alias? Either that or you need to check size too.

[1] https://github.com/mozilla/addons-validator/blob/master/src/io/utils.js#L21